locale/Makefile

```diff
@@ -44,34 +50,24 @@ uniq-po:
 	done
 
 tx-pull: $(EDITFILES)
-	tx pull -f
+	# Initialize new languages
+	cd .. && tx pull -f --all --minimum-perc 50
+	# Force update all existing languages
+	cd .. && tx pull -f --minimum-perc 0
 	for f in $(EDITFILES) ; do \
 		sed -i 's/^\("Project-Id-Version: \).*$$/\1$(DOMAIN) $(VERSION)\\n"/' $$f; \
 	done
 
 tx-update: tx-pull
 	@echo
-	@echo Run rake plugin:gettext[$(DOMAIN)] and rake plugin:po_to_json[$(DOMAIN)] from the Foreman installation, then make -C locale po-files to finish
+	@echo Run rake plugin:gettext[$(DOMAIN)] from the Foreman installation
+	@echo then run rake plugin:po_to_json[$(DOMAIN)] from the Foreman installation
+	@echo then run make -C locale mo-files to finish
 	@echo
 
-commit-translation-files: $(POFILES)
-	git add $(POFILES) $(POTFILE) $(ACTIONFILE) ../app/assets/javascripts/katello/locale
+mo-files: $(MOFILES)
+	git add $(POFILES) $(POTFILE) $(JSFILES) ../locale/*/LC_MESSAGES
```

Comment:
The `../locale/*/LC_MESSAGES` part adds the `.mo` files and I think we ignore those here.